src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -335,21 +339,26 @@ bool ZoneWindow::Init(IZoneWindowHost* host, HINSTANCE hinstance, HMONITOR monit
 
 IFACEMETHODIMP ZoneWindow::MoveSizeEnter(HWND window, bool dragEnabled) noexcept
 {
+    if (m_windowMoveSize)
+    {
+        return E_INVALIDARG;
+    }
+
     if (m_host->isMakeDraggedWindowTransparentActive())
     {
+        RestoreOrginalTransparency();
+
+        draggedWindowExstyle = GetWindowLong(window, GWL_EXSTYLE);
+
         draggedWindow = window;
         SetWindowLong(window,
                       GWL_EXSTYLE,
                       GetWindowLong(window, GWL_EXSTYLE) | WS_EX_LAYERED);

```

Comment:
`GetWindowLong(window, GWL_EXSTYLE)` -> `draggedWindowExstyle `?

Reply:
https://docs.microsoft.com/en-us/previous-versions/ms960886(v%3Dmsdn.10)

What is your naming proposition?

Reply:
I think he meant just to use variable `draggedWindowExstyle` here instead of calling `GetWindowLong` again ?